### PR TITLE
Web Onboarding NO - Combo

### PR DESCRIPTION
**Pull request by hugokallstrom**
This story will replace "Web Onboarding NO - English Combo" and "Web Onboarding NO - Norwegian Combo". All hard coded strings have been replaced with string keys, which means the story can be used for locales.

### CODE REVIEW

File: angel-data/Web Onboarding NO - Combo.json [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+  "keywords": {
+    "backButton": "Back",
+    "externalInsuranceProviderConfirmTitle": "",
+    "selectActionSelectLabel": "Select",
+    "tooltipModalInformationLabel": "Information"
+  },
+  "lastUpdate": "2022-01-31T12:31:26.248Z",
+  "locales": [
+    "en_NO"
+  ],
```
Comment (robinandeer) [inline angel-data/Web Onboarding NO - Combo.json:13]: I'm not sure if this matters but should this now also include `nb_NO`?

File: angel-data/Web Onboarding NO - Combo.json [hunk 2]
```diff
@@ -0,0 +1,393 @@
+{
+  "id": "5696df77-6e1f-42ff-9585-b3b3ff64ac0c",
+  "ifid": "D3AACABF-A05A-4221-BACB-46035E945B9A",
+  "keywords": {
+    "backButton": "Back",
+    "externalInsuranceProviderConfirmTitle": "",
+    "selectActionSelectLabel": "Select",
+    "tooltipModalInformationLabel": "Information"
```
Comment (robinandeer) [inline angel-data/Web Onboarding NO - Combo.json:8]: These need to be translated as well, right?

I guess it's:

```suggestion
    "backButton": "{STORY_KEYWORDS_BACK_BUTTON}",
    "selectActionSelectLabel": "{STORY_KEYWORDS_SELECT_ACTION_LABEL}",
    "tooltipModalInformationLabel": "{STORY_KEYWORDS_TOOLTIP_MODAL_INFORMATION_LABEL}"
```

Just make sure we've added NO translations for them :)

**Review by robinandeer — COMMENTED** [review]
Awesome, just think we should translate the story keywords as well :)

File: angel-data/Web Onboarding NO - Combo.json (hunk 2 above)
Comment (hugokallstrom) [reply angel-data/Web Onboarding NO - Combo.json:8]: Super, thank you

File: angel-data/Web Onboarding NO - Combo.json [hunk 3]
```diff
@@ -0,0 +1,393 @@
...
+  "ifid": "D3AACABF-A05A-4221-BACB-46035E945B9A",
+  "keywords": {
+    "backButton": "{STORY_KEYWORDS_BACK_BUTTON}",
+    "selectActionSelectLabel": "{STORY_KEYWORDS_SELECT_ACTION_LABEL}",
+    "tooltipModalInformationLabel": "{STORY_KEYWORDS_TOOLTIP_MODAL_INFORMATION_LABEL}"
+  },
+  "lastUpdate": "2022-01-31T12:31:26.248Z",
+  "locales": [
+    "en_NO",
+    "nb_NO
```
Comment (robinandeer) [inline angel-data/Web Onboarding NO - Combo.json:12]: oops, missing a closing quotation mark

```suggestion
    "nb_NO"
```

**Review by robinandeer — APPROVED** [review]
Looks good

**Pull request merged**